Source/common/SNTConfigurator.h

```diff
@@ -520,6 +522,12 @@
 ///
 @property(readonly, nonatomic) BOOL enableBackwardsCompatibleContentEncoding;
```

Comment:
I think we should remove this property and the key that it refers to, and add a note in the release notes that it's gone.

Reply:
Works for me.